### PR TITLE
Hubs and stores

### PR DESCRIPTION
**Pull request by getbez**
@yareda @asuraphel  hey guys please review the changes

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -555,14 +555,19 @@
   create_table "stores", force: :cascade do |t|
     t.string   "name",              null: false
     t.boolean  "temporary"
+<<<<<<< HEAD
```
Comment (asuraphel) [inline db/schema.rb]: Conflicts...

File: app/views/warehouses/index.html.erb [hunk 2]
```diff
@@ -12,20 +12,22 @@
   </div>
  <div class="col-sm-8">
     <div class="title-action">
-      <%= link_to 'New Warehouse', hubs_path, :class => "btn btn-primary"%>
+      <%= link_to 'New Warehouse',  new_hub_warehouse_path(0), :class => "btn btn-primary"%>
```
Comment (asuraphel) [inline app/views/warehouses/index.html.erb:15]: Is the `0` intentional? Just curious...

File: app/views/hubs/show.html.erb [hunk 3]
```diff
@@ -1,2 +1,186 @@
...
+        </dl>
+        <div class="row">
+          <div class='col-xm-8'></div>
+          <div class='col-xm-4'>
+            <div class="title-action">
+              <%= link_to 'Edit', edit_hub_path(@hub), :class=>'btn btn-primary' %>
+            </div>
+          </div>
+        </div>
+        <!--
```
Comment (asuraphel) [inline app/views/hubs/show.html.erb]: Please delete commented out code. Git will track it.

**Review by asuraphel — CHANGES_REQUESTED** [review]

File: app/views/warehouses/index.html.erb (hunk 2 above)
Comment (getbez) [reply app/views/warehouses/index.html.erb:15]: yap. unless we remove the hierarchy, it doesn't accept warehouses with hubs. i set that in the controller for requests directly from warehouses page

File: app/views/hubs/show.html.erb (hunk 3 above)
Comment (getbez) [reply app/views/hubs/show.html.erb]: I know git does :) was trying different looks for the page n wanted u guys to comment on it seems like i forgot to remove it . thanks though.

**Pull request merged**